coqbot-request-stamp

```diff
@@ -1 +1 @@
-DUMMY
+<PR Number / Issue Number / Comment Id>
```

Comment:
That's not clear to me. Is this all on the same line?

Reply:
This is whatever you want to see on the first line of the response to coqbot.  Whatever coqbot needs to know to know where to post the response.  Whether that's a PR number or the id of a comment to modify or both or something else altogether.  Or even empty, if all the required information is in the URL